packages/fxa-payments-server/package.json

```diff
@@ -5,13 +5,14 @@
   "scripts": {
     "lint": "npm-run-all --parallel lint:*",
     "lint:eslint": "eslint .",
-    "lint:tsc": "tsc --noEmit",
```

Comment:
eslint can handle Typescript with the config changes here, so no need to try a dry-run compiling things to surface errors